e2echart/test-risk-analysis.html

```diff
@@ -0,0 +1,107 @@
+<html lang="en">
+<head>
+    <title>Risk Analysis</title>
+    <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
+    <meta name="description"
+          content="Risk analysis is performed by Sippy to attempt to determine if the failures in this job are
+          abnormal when compared to results for similar jobs over the past week, and amidst on-going incidents
+          in the CI infrastructure. Risk analysis API will not catch everything and is a relatively simple
+          implementation today, please reach out to the Technical Release Team if you spot abnormalities or
+          have suggestions.">
+</head>
```

Comment:
Can we get the text all on one line? Seems to not like the white space.

![image](https://user-images.githubusercontent.com/429763/200684546-9e265fec-0514-4a76-a39e-34d6a7ebad2a.png)